mlflow/genai/evaluation/utils.py

```diff
@@ -36,7 +37,7 @@ def _convert_to_legacy_eval_set(data: "EvaluationDatasetTypes") -> dict:
     column_mapping = {
         "inputs": "request",
         "outputs": "response",
-        "trace": "trace",
+        "traces": "trace",
```

Comment:
```suggestion
        "trace": "trace",
```
Let's revert this change to keep `trace` as a single allowed format